.github/workflows/playwright.yml

```diff
@@ -104,17 +141,30 @@ jobs:
           cd "$(ls -1)"
           mv ./* "$src/dist/"
 
+      - name: Start Grafana Custom Location
+        if: ${{ inputs.docker-compose-file }}
+        run: |
+          docker compose -f ${{ inputs.docker-compose-file }} pull
+          GRAFANA_VERSION=${{ matrix.GRAFANA_IMAGE.VERSION }} GRAFANA_IMAGE=${{ matrix.GRAFANA_IMAGE.NAME }} docker compose -f ${{ inputs.docker-compose-file }} up -d
+
       - name: Start Grafana
+        if: ${{ !inputs.docker-compose-file }}
```

Comment:
Nit: I would merge these two steps (`Start Grafana Custom Location` and `Start Grafana`) into one and just conditionally add `-f` or not depending on the value of `inputs.docker-compose-file`, something like this:

```yaml
      - name: Start Grafana
        run: |
          $dcf="${{ inputs.docker-compose-file }}"
          # add the -f argument only if "inputs.docker-compose-file" is defined
          $dc="docker compose ${dcf:+-f "$dcf"}"
          $dc pull
          GRAFANA_VERSION=${{ matrix.GRAFANA_IMAGE.VERSION }} GRAFANA_IMAGE=${{ matrix.GRAFANA_IMAGE.NAME }} $dc up -d
```